Source/objects.cpp

```diff
@@ -4528,10 +4530,12 @@ void OperateObject(int pnum, int i, BOOL TeleFlag)
 		OperateArmorStand(pnum, i, sendmsg);
 		break;
 	case OBJ_GOATSHRINE:
-		OperateGoatShrine(pnum, i, LS_GSHRINE);
+		if(!sgOptions.Gameplay.bDisableCripplingShrines)

```

Comment:
This approach may sound like a bug because there would be no feedback for the users.

I suggest removing the boolean check here and move it to the OperateShine method instead (since both OperateGoatShrine and OperateCauldron call it).

```
void OperateShrine(int pnum, int i, int sType)
{
	...
	if (!deltaload) {
		PlaySfxLoc(sType, object[i]._ox, object[i]._oy);
		object[i]._oAnimFlag = 1;
		object[i]._oAnimDelay = 1;
	} else {
		object[i]._oAnimFrame = object[i]._oAnimLen;
		object[i]._oAnimFlag = 0;
	}

	if (ShrineIsCrippling(i)) && sgOptions.Gameplay.bDisableCripplingShrines) {
		InitDiabloMsg("Cowardice is also a sign of widsom.");
		return;
	}
	...
```

Reply:
This changes the game mechanics because people would now consider clicking on goat shrines and cauldrons.

Not being able to click on them is not a bug, this is what all reasonable players would do, so they would not notice any difference.

I am open to displaying a message or something, but you need
1) to disable goat shrines and cauldrons as well, and
2) allow players in the same game who have not disabled to the shrines to click on them, even if the other player has already tried to click.

I think a message is not strictly necessary, however.

Reply:
@Eider-McDuck yeah, you are right, so maybe only disabling and showing the message to avoid more interaction.